graphx/pom.xml

```diff
@@ -45,9 +45,14 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.jblas</groupId>
-      <artifactId>jblas</artifactId>
-      <version>${jblas.version}</version>
+      <groupId>com.github.fommil.netlib</groupId>
+      <artifactId>core</artifactId>
+      <version>1.1.2</version>
+    </dependency>
+    <dependency>
+      <groupId>net.sourceforge.f2j</groupId>
+      <artifactId>arpack_combined_all</artifactId>
+      <version>0.1</version>
```

Comment:
Some netlib-java BLAS routines contains `org.netlib.intW`, which is not part of `com.github.fommil.netlib` but `arpack_combined_all`. In the `netlib:core` pom file (https://repo1.maven.org/maven2/com/github/fommil/netlib/core/1.1.2/core-1.1.2.pom), `arpack_combined_all` is specified as a dependency but I cannot run the tests without explicitly specifying in the pom. Maybe it is because that it also include `arpack_combined_all:javadoc` as a dependency, sbt/maven doesn't resolve the dependency correctly.

Breeze already depends on netlib-java and arpack_combined_all. Should we already have something under LICENSE?